classes/class-p4-custom-taxonomy.php

```diff
@@ -200,7 +200,8 @@ public function trigger_rewrite_rules( $term_id, $tt_id, $taxonomy ) {
 		}
 
 		/**
-		 * Save custom taxonomy for planet4 post types according to the categories that were assigned to the post.
+		 * Add first term of the taxonomy to the post if the post has not any taxonomy's terms assigned to it.
+		 * Assign only the first term, if more than one terms are assigned to the post.
 		 *
 		 * @param int $post_id Id of the saved post.
```

Comment:
@kirdia Can you please add the second param of function in Docblock. (It was missed in my last changes)